docs/src/main/sphinx/sql/create-materialized-view.rst

```diff
@@ -0,0 +1,90 @@
+========================
+CREATE MATERIALIZED VIEW
+========================
+
+Synopsis
+--------
+
+.. code-block:: text
+
+    CREATE [ OR REPLACE ] MATERIALIZED VIEW
+    [ IF NOT EXISTS ] view_name
+    [ COMMENT string ]
+    [ WITH properties ]
+    AS query
+
+Description
+-----------
+
+Create a new materialized view of a :doc:`select` query. The view is a physical
+manifestation of the query results at time of creation. The data is stored and
+can be referenced by future queries. Since the data is stored directly, these
+queries are typically faster than retrieving data from a view created with the
+same query.
+
+However the data in the materialized becomes out of sync with the underlying
+data in the source tables, when that data changes. The data in the materialized
+view is updated with the :doc:`refresh-materialized-view` statement.
+
+The optional ``OR REPLACE`` clause causes the materialized view to be replaced
+if it already exists rather than raising an error.
+
+The optional ``IF NOT EXISTS`` clause checks for availability of a materialized
+view of the same name and skips creation and execution of the query if a
+materialized view exists.
+
+The optional ``COMMENT`` clause causes a ``string`` comment to be stored with
+the metadata about the materialized view. It can be displayed with the
+:doc:`show-create-materialized-view` statement and is available in
+``system.metadata.materialized_view_properties``.
+
+The optional ``WITH`` clause is used define properties to use for the
+materialized view creation. Separate multiple property/value pairs by commas.
+The supported properties are different for each connector and detailed in the
+SQL support section of the specific connector documentation.
+
+Examples
+--------
+
+Create a simple materialized view ``cancelled_orders`` over the ``orders`` table
```

Comment:
Personal preference: The following statement creates

Reply:
this is existing style in all the other language docs so I am staying consistent for now .. but I agree with your preference.